.golangci.yaml

```diff
@@ -0,0 +1,52 @@
+run:
+  timeout: 5m
+  skip-files:
+    - pkg/target/matchcrd_constant.go
```

Comment:
same here

Reply:
Sure, this was carry over, we can remove.